### DISCUSSION TITLE
This UI is about the worst I've ever encountered.

### DISCUSSION THREAD

**Discussion by GWHAYWOOD** [community/community#133780] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

It said "Wait a moment" and started a little twirly thing.

That was about half an hour ago, and I'm still waiting.

What a steaming pile of cr@p.

**Comment by akibhaider** [comment]
Same here 🙌

**Reply by kelechi055** [reply]
Same
